### ISSUE TITLE
Checkpoint/Restart is not working.

### ISSUE THREAD

**Issue by pagrubel** [lanl/BEE#750]
Using the workflow in beeflow/data/cwl/bee_workflows/clamr-wf-checkpoint that worked previously. Querying the status of the workflow gives:
```beeflow query 17
Running
clamr--TIMEOUT
ffmpeg--WAITING
```

The TIMEOUT state should spawn restart tasks.

**Comment by pagrubel** [comment]
These are the entries at the end of the task manager log:
```[2023-12-04 17:22:44,426] beeflow.task_manager:submit_jobs(): Job Submitted clamr: job_id: 15569181 job_state: PENDING
[2023-12-04 17:24:09,374] beeflow.task_manager:update_jobs(): state: TIMEOUT
[2023-12-04 17:24:09,375] beeflow.task_manager:update_jobs(): TIMELIMIT/TIMEOUT task_checkpoint: {'file_path': 'checkpoint_output', 'num_tries': '3', 'container_path': 'checkpoint_output', 'enabled': 'True', 'restart_parameters': '-R', 'file_regex': 'backup[0-9]*.crx'}
[2023-12-04 17:24:09,544] beeflow.task_manager:update_task_state(): WFM not responding when sending task update.```

**Comment by aquan9** [comment]
Steven fixed the retries type issue. It sounds like we need to go to wf_update.py.  The workflows are getting stuck in the timeout state when they run out of retries, but they should move to the "fail" state. Currently the branch he has to fix the issue is working to do the restart, but it just gets stuck on this last state update.

**Comment by pagrubel** [comment]
Addressed in #776
